src/sonic-config-engine/sonic-cfggen

```diff
@@ -15,6 +15,7 @@ Examples:
 See usage string for detail description for arguments.
 """
 
+from __future__ import print_function
```

Comment:
What's it for? Print to stderr?

Reply:
To write 'print' in the most compatible style, for example python3.

---
In reply to: [183192730](https://github.com/Azure/sonic-buildimage/pull/1631#discussion_r183192730) [](ancestors = 183192730)